Remotion/Web/Development.WebTesting.IntegrationTests/ScreenshotTest.cs

```diff
@@ -661,6 +667,32 @@ public void ScreenshotTransformationTest ()
       Helper.RunScreenshotTestExact<ScreenshotTest> (PrepareTest(), ScreenshotTestingType.Browser, test);
     }
 
+    private void RetryTest (Action action, int times)
+    {
+      Exception lastThrownException = null;
+
+      for (int i = 0; i < times; i++)
+      {
+        try
+        {
+          action();
+        }
+        catch (AssertionException ex)
+        {
+          lastThrownException = ex;
+          continue;
+        }
+
+        lastThrownException = null;
+        break;
+      }
+
+      if (lastThrownException != null)
+      {
+        throw lastThrownException;

```

Comment:
doing it this way kills our stacktrace. code needs to be rewritten to rethrow immediately if the max number of tries is exceeded. in theory, we could also do a when-clause on the catch block. Otherwise, it's just an IF and a THROW w/o the exception variable.